src/Config.php

```diff
@@ -130,6 +106,7 @@ public function getBaseUrl(): string
 
     /**
      * Na základě nastavení vrací argumenty pro SoapClient
+     * Neumožňujeme uživateli primo modifikovat options aby byly vzdy validni a kompatibilni se Skautis API
      *
```

Comment:
Asi out of scope, ale IMO tahle metoda do Config objektu vůbec nepatří a měla by být přímo v místě, kde se ten Soap používá.

Reply:
Taky jsem nad tim premyslel, ale zatim jsem se soustredil na to aby byl config "bezpecnejsi" na pouziti.

Reply:
Ok